Square_test.py

```diff
@@ -0,0 +1,75 @@
+import json
+import argparse
+
+def bubblesort(my_list):
+    for i in range (0, len(my_list) - 1):
+        for j in range(0, len(my_list) -1 -i):
+            if my_list[j] > my_list[j+1]:
+                my_list[j], my_list[j+1] = my_list[j+1], my_list[j]
+    return my_list
+parser = argparse.ArgumentParser('Input Coordinate File')
+parser.add_argument('-r', '--read', required=True, type=str, help= 'enter file to be read')
+args, unknown = parser.parse_known_args()
+

```

Comment:
Try to execute that program with the file name that does not exist.
You need to start thinging not only the "goldem path", but also errors / exceptions.